src/app/modules/form/admin-form/admin-form.controller.ts

```diff
@@ -1561,10 +1563,10 @@ export const submitEmailPreview: ControllerHandler<
   }
   const form = formResult.value
 
-  const parsedResponsesResult =
-    await EmailSubmissionService.validateAttachments(responses).andThen(() =>
-      ParsedResponsesObject.parseResponses(form, responses),
-    )
+  const parsedResponsesResult = await SubmissionService.validateAttachments(
+    responses,
+    form.responseMode === FormResponseMode.Email,
```

Comment:
/nit: hmmmm i would personally prefer this to be Enum instead of boolean. there could be more than two types of responseMode in future

Reply:
That's true, refactored to use enums in [64765de](https://github.com/opengovsg/FormSG/pull/6595/commits/64765de3d5d64fd3e8622fcb27fdd85574dda230), thanks!